### PR TITLE
documentation enhancements

### PR DESCRIPTION
**Pull request by ByReaL**
### Description of PR

Summary:
Fixes # 
- improved server configuration steps and updates for ubuntu 20.04
- modernized testbed config docs
- 28 times password.txt was used and 96 times .password in examples, made all docs use .password file for consistency of the commands and examples.

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Docs
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
while using the docs to create my own setup I enhanced the details needed for the steps
Ubuntu 20.04 support
consistency in sample commands

#### How did you do it?
I have created my own setup and documented the steps taken and enhanced the docs where more details were needed.

#### How did you verify/test it?
I have done the steps described in the doc, in the order from the doc

#### Any platform specific information?
added interface config using netplan for ubuntu 20.04 but also kept the old instructions for interfaces for ubuntu 18.04

### Documentation 

this PR is only for documentation updates.

### CODE REVIEW

File: ansible/doc/README.testbed.Setup.md [hunk 1]
```diff
@@ -96,15 +162,23 @@ Once you are in the docker container, you need to modify the testbed configurati
         ```
 
 - VMs
-    - Download [vEOS image from Arista](https://www.arista.com/en/support/software-download).
+    - Download [vEOS-lab image from Arista](https://www.arista.com/en/support/software-download).
     
     - Copy these image files to `~/veos-vm/images` on your testbed server:
         - `Aboot-veos-serial-8.0.0.iso`
-        - `vEOS-lab-4.20.15M.vmdk`
+        - `vEOS-lab-4.25.0F.vmdk`
```
Comment (lguohan) [inline ansible/doc/README.testbed.Setup.md:169]: my recommendation is not change this until you have validated all the tests with this vEOS image?

File: tests/kvmtest.sh [hunk 2]
```diff
@@ -14,11 +14,11 @@ chmod 600 .ssh/id_rsa
 
 # Refresh dut in the virtual switch topology
 cd $SONIC_MGMT_DIR/ansible
-./testbed-cli.sh -m $inventory -t $testbed_file refresh-dut $tbname password.txt
+./testbed-cli.sh -m $inventory -t $testbed_file refresh-dut $tbname .password
```
Comment (lguohan) [inline tests/kvmtest.sh:17]: the test is broken, can you check?

**Comment by lguohan** [comment]
the change to kvmtest.sh breaks the pr test.

File: ansible/doc/README.testbed.Setup.md (hunk 1 above)
Comment (ByReaL) [reply ansible/doc/README.testbed.Setup.md:169]: i need to revert this, tested and did not work.

File: tests/kvmtest.sh (hunk 2 above)
Comment (ByReaL) [reply tests/kvmtest.sh:17]: maybe that file does not exist on the test setup?

Comment (ByReaL) [reply tests/kvmtest.sh:17]: 1) can we rename the file on the setup so it is found and test work again?
or
2) I can change all ".password" to "password.txt" but that was generating more changes.
"28 times password.txt was used and 96 times .password in examples, made all docs use .password file for consistency of the commands and examples."
for me, either way, works. The fix was intended to bring consistency.

**Comment by nirmalya-keysight** [comment]
Seems like there are a quite a few typos.

`self.password` was changed to `selfpassword.txt`

Similar for `rinfopassword.txt`.

**Pull request closed without merging**